kubernetes/config-maps/concent/admin_settings.py.j2

```diff
@@ -0,0 +1,5 @@
+ADMINS = [
+{% for admin in admins %}
+    ("{{ admin.name }}", "{{ admin.mail }}"),
+{% endfor %}
```

Comment:
This is really a part of the e-mail settings. Please merge those two files.

Reply:
Done.